gap/OscarInterface/gap/perf.grp.gi

```diff
@@ -0,0 +1,40 @@
+BindGlobal( "PerfGrpLoadOscar", function(sz)
```

Comment:
Replacing `PerfGrpLoad` like that is a bit brittle :-/.

Here's an alternative idea: perhaps we could always provide `perf27.grp` and `perf33.grp` files, but instead of them being symlinks, provide actual files that essentially do something like this:
```gap
Read(JuliaToGAP(IsString, JuliaEvalString("""artifact"gap_hulpke_extraperfect/extraperfect-80c0cf27bf98ba3eabd287db5ab4fd0f249e0559/perf27.grp" """)));
```

Reply:
Yeah, I thought similar. Your idea seems a bit less hacky. I tried to implement it and after some fiddling around got it to work in daa3f0a.